frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/volume-wrapper.ts

```diff
@@ -88,16 +45,25 @@ export class VolumeWrapper extends ObjectWithTypePropertyWrapper<
 
   getContainerImage = () => getVolumeContainerImage(this.data);
 
-  replaceType = (type: VolumeType, typeData: CombinedTypeData, sanitize = true) => {
-    this.setType(type, sanitize ? sanitizeTypeData(type, typeData) : typeData);
-    return this;
-  };
+  protected sanitize(
+    type: VolumeType,
+    { name, claimName, image, userData, userDataBase64 }: CombinedTypeData,
+  ): CombinedTypeData {
+    if (type === VolumeType.DATA_VOLUME) {
+      return { name };
+    }
+    if (type === VolumeType.PERSISTENT_VOLUME_CLAIM) {
```

Comment:
Is there a particular reason there's no space between these two conditionals? If not, could you please add one for consistency?

Reply:
thanks! this made me realize that switch statements are better choice